docs/es/docs/python-types.md

```diff
@@ -237,28 +272,49 @@ Una vez más tendrás todo el soporte del editor:
 
 ## Modelos de Pydantic
 
-<a href="https://pydantic-docs.helpmanual.io/" class="external-link" target="_blank">Pydantic</a> es una library de Python para llevar a cabo validación de datos.
+<a href="https://pydantic-docs.helpmanual.io/" class="external-link" target="_blank">Pydantic</a> es una libreria de Python utilizada para llevar a cabo validación de datos.
 
 Tú declaras la "forma" de los datos mediante clases con atributos.
 
 Cada atributo tiene un tipo.
 
-Luego creas un instance de esa clase con algunos valores y Pydantic validará los valores, los convertirá al tipo apropiado (si ese es el caso) y te dará un objeto con todos los datos.
+Luego creas un instancia de esa clase con algunos valores y Pydantic los validará, los convertirá al tipo apropiado (si ese es el caso) y te dará un objeto con todos los datos.
```

Comment:
Please revert the word "instancia", even though it's sometimes used in Spanish, it's actually incorrect, "instance" means "ejemplar", that's why I keep the technical term in English.